spec/factories/ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+FactoryGirl.define do
+  factory :ems_nuage_with_vcr_authentication, :parent => :ems_nuage_network do
+    zone do
+      _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+      zone
+    end
+
+    after(:build) do |ems|
+      ems.hostname = Rails.application.secrets.nuage_network.try(:[], 'host') || 'nuagenetworkhost'
+    end
+
+    after(:create) do |ems|
+      userid   = Rails.application.secrets.nuage_network.try(:[], 'userid') || 'NUAGE_USER_ID'
+      password = Rails.application.secrets.nuage_network.try(:[], 'password') || 'NUAGE_PASSWORD'
+
+      # Ensure that VCR will obfuscate the basic auth
+      VCR.configure do |c|
```

Comment:
To sum up: definetly move VCR configuration out of factory. If possible, merge with `spec_helper.rb`, if not, put directly into `refresher_spec.rb`.